src/components/SavedProjectPlan/StandardCard.jsx

```diff
@@ -0,0 +1,44 @@
+import {
+  Box,
+  Card,
+  CardBody,
+  CardHeader,
+  Flex,
+  Heading,
+  Image,
+} from "@chakra-ui/react";
+const StandardCard = () => {
```

Comment:
You mentioned adding props. that is a great idea. so first make sure to merge main to your branch. Next, look into adding default props (possibly search some documentation because I personally don't know much about this myself).

What will happen is that you don't want to have your default props be in the project plan page. Either have the default props be on this page itself or another page where you call it `defaultStandardProps.js` or something along those lines. 

In this case the default props would be, `[{title: string}, {imageUrls: array of imageUrl strings}, {criteria: string}]`

Reply:
I got the prop logic to work and tested with other images, however, it's still weird with our southface images from Next.js, so I'm using hardcoded images (fallback URLs)